waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -88,8 +88,8 @@ genNvmrc =
       -- We want to specify only the major version, check the comment in `ServerGenerator.hs` for details
       (Just (object ["nodeVersion" .= show (SV.major nodeVersion)]))
 
-npmDepsForWasp :: N.NpmDepsForWasp
-npmDepsForWasp =
+npmDepsForWasp :: AppSpec -> N.NpmDepsForWasp
+npmDepsForWasp _spec =
```

Comment:
Hm, how come we need this? Something to do with eager vs lazy?

Reply:
The suggestion and thinking was to keep it consistent with Server dependencies, even though we may not need it quite yet.

Reply:
Ok, makes sense.